runtime/doc/options.txt

```diff
@@ -635,7 +635,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	(or Vim is run inside an xterm invoked with "-cjkwidth" option.),
 	this option should be set to "double" to match the width perceived
 	by Vim with the width of glyphs in the font.  Perhaps it also has
-	to be set to "double" under CJK Windows 9x/ME or Windows 2k/XP
+	to be set to "double" under CJK Windows 2k/XP
```

Comment:
IIRC libuv doesn't support Windows 2000 (only XP and up), so the `2k` should be removed.
Also the line can be re-wrapped.